cms/jinja2/assets/js/hide-date-text-on-non-provisional-releases.js

```diff
@@ -0,0 +1,16 @@
+function toggleDateTextVisibility() {
+  var statusField = document.getElementById('id_status');
```

Comment:
Should be a `const` - `var`s should not be used anymore unless for some very specific reasons when you need a function scoped variable. Here and on line 14.
```suggestion
  const statusField = document.getElementById('id_status');
```

Reply:
Thanks for refreshing my rusty JS knowledge 😄

Reply:
[d5632aa](https://github.com/ONSdigital/dis-wagtail/pull/158/commits/d5632aa9f8ce5f1e73229f8d81c88af5c4a2424c)